generator/AWSPSGeneratorLib/HelpMaterials/Examples/DS/Register-DSCertificate.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$Certificate = gc contoso.cer -raw
+Register-DSCertificate -DirectoryId d-123456ijkl -CertificateData $Certificate
+c-906731e350</code>
```

Comment:
I wonder what is `c-906731e350` here? If it is output then, let's add a new line between the command and output.

Also, can you update the gc with full command?

```
$Certificate = Get-Content -Path contoso.cer -Raw
Register-DSCertificate -DirectoryId d-123456ijkl -CertificateData $Certificate
```

Reply:
Updated.